lib/lor/lib/router/layer.lua

```diff
@@ -5,7 +5,7 @@ local pairs = pairs
 local ipairs = ipairs
 local type = type
 local setmetatable = setmetatable
-local ostime = os.time
+local ostime = ngx.time
```

Comment:
So we need a nginx stub for mock test like [this](https://github.com/tamqm/nginx-cdn/blob/master/Projects/bsm/fakengx/fakengx.lua)?